includes/sanitizers/class-amp-gallery-block-sanitizer.php

```diff
@@ -7,6 +7,7 @@
 
 use AmpProject\AmpWP\Embed\HandlesGalleryEmbed;
 use AmpProject\Html\Tag;
+use AmpProject\Dom\Element;
```

Comment:
Nit, Not a blocker.
This could be above `use AmpProject\Html\Tag;`

Reply:
I'm not sure I understand. There is already `use AmpProject\Html\Tag`. The use of `AmpProject\Dom\Element` is due to this new line:

https://github.com/ampproject/amp-wp/blob/920eaa1ff2a03f1365cc2cc9a135b7eee181ada5/includes/sanitizers/class-amp-gallery-block-sanitizer.php#L81